### DISCUSSION TITLE
Target [Illuminate\Contracts\Debug\ExceptionHandler] is not instantiable in Laravel 11.

### DISCUSSION THREAD

**Discussion by wanadri** [laravel/framework#51233] [Q&A]
Hi there,

Currently i having some issue migrating from laravel 10 to laravel 11. It prompt an error `Target [Illuminate\Contracts\Debug\ExceptionHandler] is not instantiable` after i run composer command.

--------------------------------------------------------------------------------------------------------------

### Scenario

I run composer command like below:
`composer dump-autoload`

An error has prompt like a screenshot below:
<img width="1321" alt="Screenshot 2024-04-28 at 11 03 23 PM" src="https://github.com/laravel/framework/assets/13805716/089451cb-029a-4c43-b030-6e7832dc6ebe">

Since the error it is too general and not show which file causing the error. I try to trace by `dd` in _HandleExceptions_ class in _Illuminate\Foundation\Bootstrap_.
Example like below:
```
public function handleException(Throwable $e)
    {
        dd($e->getTrace()); // log here...

        static::$reservedMemory = null;

        try {
            $this->getExceptionHandler()->report($e);
        } catch (Exception) {
            $exceptionHandlerFailed = true;
        }
```

An it shows like screenshot below:
<img width="1064" alt="Screenshot 2024-04-28 at 11 13 42 PM" src="https://github.com/laravel/framework/assets/13805716/426d3ea1-fe5e-4464-afdb-f2afaca744ec">

and then i saw _CollisionServiceProvider_ class line 61 has been mentioned on the list.
It shows like the screenshot below:
<img width="610" alt="Screenshot 2024-04-28 at 11 23 00 PM" src="https://github.com/laravel/framework/assets/13805716/9a1227fc-a83b-4c39-8567-97e3241c26d9">

After that, i commented on that part.
<img width="569" alt="Screenshot 2024-04-28 at 11 22 09 PM" src="https://github.com/laravel/framework/assets/13805716/822eefab-a8f9-434c-bbe8-b48789efc56d">

And then, i run again the composer command `composer dump-autoload`.

No error prompt.

--------------------------------------------------------------------------------------------------------------

Need your help to encounter this issue.

Thank you.

**Comment by dmouse** [comment]
maybe too late but if you are using the new boostraping way you need to include `->withExceptions()` 

```
$app = Application::configure(basePath: dirname(__DIR__))
    ->withProviders([
    ])
    ->withExceptions() // this
    ->create();
```
